docs/Installation.md

```diff
@@ -13,7 +13,7 @@ _**NOTE:**_
         sudo easy_install pip
         sudo pip install 'cryptography==1.4' Flask launchpadlib simplejson logging
 
-* For Ubuntu (16.04, 17.04, 17.10):
+* For Ubuntu (18.04, 19.04, 20.04):
 
         sudo apt-get update
         sudo apt-get install -y python3 python3-pip gcc git python3-dev libssl-dev libffi-dev cron python3-lxml apache2 libapache2-mod-wsgi-py3
```

Comment:
Were you able to confirm the package names haven't changed at all?

Reply:
I'm running on Ubuntu 20.04. Copy pasted each command